### PR TITLE
Pass 2

### PR DESCRIPTION
**Pull request by tabbykatz**
I know this is going very slowly but it is going, so please give me your feedback when you have time.  💖

### CODE REVIEW

File: cover_letters/generic.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
 # In which I channel someone confident/ a letter to no one in particular
 
-I'm a self-taught developer apprentice ready to join a team where I can grow and help others do the same. I've been working towards this for two years of broad study and two truths have emerged:
+[aa: add personal story]
+I'm an enthusiastic new developer ready to join a team where I can grow and help others do the same. I've been working towards this for two years of broad study and two truths have emerged:
 
 - I enjoy learning and solving problems
 - I am a flexible and joyful student, apprentice, teammate
 
 If your team will supply me with technical challenges and room to grow, it is ideal for me.
```
Comment (gsong) [inline cover_letters/generic.md:9]: This sentence sounds awkward to me. I would use a more active voice to highlight the environment under which you will thrive.

File: cover_letters/generic.md [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
 
-While it is no longer unusual for humans to take non-traditional paths to software, I want to clear something up. _Self-taught doesn't mean I go it alone._ Whether from books or the internet, structured MOOCs or seminars, no one is truly _self-taught_. Certain humans come to mind; without them I would not be writing this. Most importantly, self-taught means that I've intentionally learned how to learn, and how _I_ learn. Mentorship, collaboration, and iteration in groups are vital.
+While it is no longer unusual for humans to take non-traditional paths to software, I want to clear something up. _I didn't teach myself how to code._ Whether from books or the internet, structured MOOCs or seminars, no one is truly _self-taught_. Certain humans come to mind; without them I would not be writing this. Most importantly, I've intentionally learned how to learn, and how _I_ learn. Mentorship, collaboration, and iteration in groups are vital. [to: I think this is closer to what I was trying to say, anyway]
```
Comment (gsong) [inline cover_letters/generic.md:11]: [suggest] I know this is your words, but in a professional context using "humans" in this way might be off-putting compared to "people."

Comment (gsong) [reply cover_letters/generic.md:11]: [?] I'm not quite sure what "iteration in groups" means.

File: cover_letters/generic.md [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
 
 - I love seeing the world from different viewpoints
 - It is my goal to never stop refining myself, both as an engineer and a human, and that requires challenge.
 
 I want to work with diverse people on complex projects that put me slightly outside of my comfort zone. This is where growth happens both for myself and those around me. Ideally, we will also solve some interesting problems along the way. If that sounds like what you need, I look forward to meeting you.
```
Comment (gsong) [inline cover_letters/generic.md:16]: [?] Complex or complicated?

File: resume.md [hunk 4]
```diff
@@ -4,57 +4,59 @@
...
+- A healthy company culture of mentorship with a robust feedback framework
 
 ## Technologies
 
 Proficient: Bash, C, Python, JavaScript, SQL
 
-Current focus: React, CSS, HTML Express, Node.js, PostgreSQL
+Current focus: React, CSS, HTML, Express, Node.js, PostgreSQL
+
+Familiar: Docker, Flask, Nginx, puppet, Next.js
```
Comment (gsong) [inline resume.md:22]: [fix] Capitalize Puppet.

File: resume.md [hunk 5]
```diff
@@ -4,57 +4,59 @@
...
 
 ### Techtonica, Remote (July 2021—Current)
 
-Six-months full-time immersive full-stack development education
+Six-months full-time immersive full-stack development education.
+
+Capstone: Game Night, An app for board game enthusiasts who attend in-person gaming meetups.
 
-[Capstone project placeholder]
+- Full-stack app built with postgreSQL, Express, Node.js, and React
```
Comment (gsong) [inline resume.md:34]: [fix] PostgreSQL.

File: resume.md [hunk 6]
```diff
@@ -4,57 +4,59 @@
...
-- XYZ
+My capstone project, <https://www.tabbykatz.com>, is built with Next.js and React and continuously deployed with Vercel. My blog is written in Markdown and published from the command line.
 
 Other notable activities:
 
-- Re-implemented the C standard library `printf`
-- Built a terminal shell in C
+- Re-implemented the C standard library, including `printf`, to develop thorough fundamental knowledge of the language
+- Built a terminal shell in C using custom library functions
 - Created a full stack LAMP(ython) AirBnB clone
```
Comment (gsong) [inline resume.md:55]: [?] LAMP? Do you mean Angular, Python, MySQL?

File: resume.md (hunk 4 above)
Comment (gsong) [reply resume.md:22]: [fix] Be consistent throughout with how you refer to nginx. I believe the preferred style is NGINX.

File: resume.md (hunk 6 above)
Comment (tabbykatz) [reply resume.md:55]: No. I mean https://en.wikipedia.org/wiki/LAMP_(software_bundle)

**Pull request merged**